### PR TITLE
Add a new resource template to create a Linux VM with new Azure Linux Agent

### PR DESCRIPTION
**Pull request by SuperScottz**

### CODE REVIEW

File: vm-linuxagent-upgrade/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,197 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters" : {
+        "newStorageAccountName": {
+            "type": "string",
+            "defaultValue" : "uniquestorageaccountName"
```
Comment (singhkays) [inline vm-linuxagent-upgrade/azuredeploy.json:7]: Can you add metadata description tags to the parameters?

File: vm-linuxagent-upgrade/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        "virtualNetworkName":{
+            "type" : "string",
+            "defaultValue" : "myVNET"
+        },
+        "nicName":{
+            "type" : "string",
+            "defaultValue":"myNIC"
+        },
+		"LinuxAgentVersion":{
+81          "type" : "string",
```
Comment (singhkays) [inline vm-linuxagent-upgrade/azuredeploy.json:48]: Are these numbers a copy paste typo?

**Comment by singhkays** [comment]
@SuperScottz Left comments. We're moving to the new repo https://github.com/Azure/azure-quickstart-templates

It'll be great if you could address the comments and send a PR to the new repo

File: vm-linuxagent-upgrade/azuredeploy.json (hunk 1 above)
Comment (SuperScottz) [inline vm-linuxagent-upgrade/azuredeploy.json:7]: thank you for reminder,  just updated.

File: vm-linuxagent-upgrade/azuredeploy.json (hunk 2 above)
Comment (SuperScottz) [inline vm-linuxagent-upgrade/azuredeploy.json:48]: it's a error from copying the final script, have created it.

**Comment by SuperScottz** [comment]
@singhkay Sure, I will send PR to that new repo with updated files.

**Pull request closed without merging**